src/App.js

```diff
@@ -1,19 +1,16 @@
 import React, { Fragment, Component } from 'react'
-import Welcome from './components/welcome.js'
+import Welcome from './components/Welcome.js'
 
 export default class App extends Component {
```

Comment:
You can convert this component to be stateless / functional since you don't need the state.